cg/store/models.py

```diff
@@ -86,43 +127,45 @@ def low_priority(self):
         return self.priority_int < 1
 
 
-class Application(Model):
+class Application(Base):
     __tablename__ = "application"
 
-    id = Column(types.Integer, primary_key=True)
-    tag = Column(types.String(32), unique=True, nullable=False)
-    prep_category = Column(types.Enum(*PREP_CATEGORIES), nullable=False)
-    is_external = Column(types.Boolean, nullable=False, default=False)
-    description = Column(types.String(256), nullable=False)
-    is_accredited = Column(types.Boolean, nullable=False)
-
-    turnaround_time = Column(types.Integer)
-    minimum_order = Column(types.Integer, default=1)
-    sequencing_depth = Column(types.Integer)
-    min_sequencing_depth = Column(types.Integer, default=0, nullable=False)
-    target_reads = Column(types.BigInteger, default=0)
-    percent_reads_guaranteed = Column(types.Integer, nullable=False)
-    sample_amount = Column(types.Integer)
-    sample_volume = Column(types.Text)
-    sample_concentration = Column(types.Text)
-    sample_concentration_minimum = Column(types.DECIMAL)
-    sample_concentration_maximum = Column(types.DECIMAL)
-    sample_concentration_minimum_cfdna = Column(types.DECIMAL)
-    sample_concentration_maximum_cfdna = Column(types.DECIMAL)
-    priority_processing = Column(types.Boolean, default=False)
-    details = Column(types.Text)
-    limitations = Column(types.Text)
-    percent_kth = Column(types.Integer, nullable=False)
-    comment = Column(types.Text)
-    is_archived = Column(types.Boolean, default=False)
-
-    created_at = Column(types.DateTime, default=dt.datetime.now)
-    updated_at = Column(types.DateTime, onupdate=dt.datetime.now)
-
-    versions = orm.relationship(
-        "ApplicationVersion", order_by="ApplicationVersion.version", back_populates="application"
+    id: Mapped[intpk]
+    tag: Mapped[str32] = mapped_column(unique=True)
+    prep_category: Mapped[PrepCategory]
+    is_external: Mapped[bool] = mapped_column(default=False)
+    description: Mapped[str256]
+    is_accredited: Mapped[bool] = mapped_column(nullable=False)
```

Comment:
isn't nullable set to False for default if no `Optional[]` argument is added to the `Mapped`? I think it would be ok to be explicit but then we should be consistent in all the fields. I would prefer omitting it and adding only `nullable=True` whenever is needed

Reply:
Good spot. I had just missed removing it